### ISSUE TITLE
Password handling issue in 'config.ini'

### ISSUE THREAD

**Issue by Gofannon** [Blount/LBCAlerte#63]
System
-----------

Server runs Yunohost with LBCAlerte version 3.3 : https://github.com/YunoHost-Apps/LBCAlerte_ynh

Description
-------------

My smtp password has a pattern which is leading to the message "Un problème est survenu lors de l'exécution du programme."

How to reproduce
----------------------

1. Go to  `Admininistration > MAIL`
2. Fill all the fields
3. In the password, put `hello"/'there` (my password has this pattern)
4. Click "Save"
5. Error "Un problème est survenu lors de l'exécution du programme. " shows up

**Logs**
```
tail var/log/error.log
2017-06-03T20:07:31+02:00 ERROR Config_Lite_Exception_UnexpectedValue : #0 key not found, no default value given. (/var/www/LBCAlerte/lib/Config/Lite.php:364)
```

How to get back the MAIL panel
----------------------------------------

1. Edit the file  `var/config.ini`
2. Restore the field `smtp[password] = ""`
3. Go back to the web interface
4. Change email address and/or use a password without this pattern

Other problematic pattern
----------------------------------------

If using the password `hello"there`, this is no error and I get a white blanck screen. And all the admin area does not run anymore.

**Comment by Blount** [comment]
This will fixed in the next version.
Thank you to reported this issue.

**Comment by Blount** [comment]
Fixed by 5cd0b99
